docs/ocm-integration.md

```diff
@@ -242,3 +242,141 @@ ocm get subscriptions --parameter search="status = 'Active'"
+index 6e909a7a..ebaee754 100644
+--- a/pkg/api/capability_types.go
++++ b/pkg/api/capability_types.go
+@@ -28,6 +28,7 @@ const CapabilitySubscribedOcpMarketplace = "capability.cluster.subscribed_ocp_ma
+ const CapabilitySubscribedOsdMarketplace = "capability.cluster.subscribed_osd_marketplace"
+ const CapabilityEnableTermsEnforcement = "capability.account.enable_terms_enforcement"
+ const CapabilityBareMetalInstallerAdmin = "capability.account.bare_metal_installer_admin"
++const CapabilityBareMetalInstallerMultiarch = "capability.account.bare_metal_installer_multiarch"
+ const CapabilityReleaseOcpClusters = "capability.cluster.release_ocp_clusters"
+ const CapabilityAutoscaleClustersDeprecated = "capability.organization.autoscale_clusters"
+ const CapabilityAutoscaleClusters = "capability.cluster.autoscale_clusters"
+@@ -60,6 +61,7 @@ var CapabilityKeys = []string{
+        CapabilitySubscribedOcpMarketplace,
+        CapabilitySubscribedOsdMarketplace,
+        CapabilityBareMetalInstallerAdmin,
++       CapabilityBareMetalInstallerMultiarch,
+        CapabilityReleaseOcpClusters,
+        CapabilityEnableTermsEnforcement,
+        CapabilityUseRosaPaidAMI,
+```
+
+That needs to be merged, and then the new version of AMS pushed to the
+corresponding environment (integration, stage and eventually production). The
+SD-B team is responsible for that, ping the `@sd-b-team` in the
+[#service-development-b](https://coreos.slack.com/archives/CBDNMS43V) channel
+in Slack.
+
+The second step is to enable the capability for the organizations that you
+want, so you need to find the identifier of that organization. For example, for
+your own organization. For example, for yourself:
+
+```shell
+$ ocm whoami | jq -r '.organization.id'
+2GOuuT2l3odeHRdWqY57fviSE2K
+```
+
+And for a given user name:
+
+```shell
+$ ocm get /api/accounts_mgmt/v1/accounts -p search="username = 'jane.doe'" | jq -r '.items[].organization.id'
+2GOuxr72XcFbrWvyA9rS9McOUjO
+```
+
+Then add the label to the organization (labels are the mechanism used by AMS to
+store these capabilities):
+
+```shell
+$ ocm post /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels <<.
+{
+  "key": "capability.organization.bare_metal_installer_multiarch",
+  "value": "true",
+  "internal": true
+}
+.
+```
+
+To disable a capability just delete the label:
+
+```shell
+$ ocm delete /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels/capability.organization.bare_metal_installer_multiarch
+```
+
+To check the capabilities enabled for an organization:
+
+```shell
+$ ocm get /api/accounts_mgmt/v1/organizations/2GOuxr72XcFbrWvyA9rS9McOUjO/labels | jq -r '.items[] | select(.type == "Organization") | .key'
+sts_ocm_role
+capability.organization.bare_metal_installer_multiarch
+capability.organization.hypershift
+capability.cluster.subscribed_ocp
+moa_entitlement_expires_at
+```
+
+Chances are that you will not have permission to enable or disable
```

Comment:
regarding the permission, please add that the user should have the BareMetalInstallerAdmin for integration & staging, and for production we should open a ticket